### DISCUSSION TITLE
apostar automatico

### DISCUSSION THREAD

**Discussion by farofakids** [elizandrodantas/bot-blaze-telegram#8] [Ideas]
É possivel realizar as apostas automaticamente usando o chromedriver?
A proposito parabens pelo excelente trabalho.

**Comment by elizandrodantas** [comment]
Não sei dizer mano, mas por request sim. Com duas request é possível fazer isso: 

1. Login
2. Aposta

Tenho um modelo aqui, com as request.

Vlw..!
